active/website.md

```diff
@@ -0,0 +1,61 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Members triage the project’s issues and pull requests.
+- Member maintain and monitor the website including updating versions.
+- Mentor new contributors to the website.
+- Run or support djangoproject.com sessions in DjangoCon sprints.
+- Chair, Co-Chair and Board Liaison can sign off on new features.
+
+## Initial membership
+
+- Chair: Sarah Abderemane
+- Co-Chair: Saptak Sengupta
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Sarah Abderemane
+- Other members:
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+
+
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django and should be able to work with Django. Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
+
+### How do people who want to join sign up / volunteer / express interest?
+Individuals can express interest by emailing to the working group mailing list at `website-wg@djangoproject.com` 
+
+### How will decisions on adding/removing members be handled?
+Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members. The WG will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes.
+
+Members join the group for one year term. At the end of this term, they need to opt into staying involved to keep being 
+a member of the group.
+
+
+## Budget
+No allocated budget
+
+## Comms
+- Private channel in the DSF slack
```

Comment:
```suggestion
- A channel in the Django discord server, accessible only to DSF mrmbers
```
It would be possible to use a subsection of the DSF-Members in the Discord server? It's already there, and it will help to interact with other WGs (eg. accessibility, social, ...)

Reply:
> I'm confused, according Thibaud's answer in the thread below it's should be a DSF slack channel
Yes, you are right. I only realized that the Discord is already there. Using Slack will add, another tool to install and use in addition to Django Forum, Discord server, etc ... We risk spending a lot of time and focus only on checking all these tools and switching between them.
But, we already agreed on Slack so I'll close the suggestion.